public/js/services/game.js

```diff
@@ -112,21 +110,24 @@ angular.module('mean.system')
       if (data.table.length === 0) {
         game.table = [];
       } else {
-        var added = _.difference(_.pluck(data.table,'player'), _.pluck(game.table,'player'));
-        var removed = _.difference(_.pluck(game.table,'player'), _.pluck(data.table,'player'));
+        const added = _.difference(_.pluck(data.table, 'player'), _.pluck(game.table, 'player'));
+        const removed = _.difference(_.pluck(game.table, 'player'), _.pluck(data.table, 'player'));
         for (i = 0; i < added.length; i++) {
-          for (var j = 0; j < data.table.length; j++) {
+          for (let j = 0; j < data.table.length; j++) {
             if (added[i] === data.table[j].player) {
-              game.table.push(data.table[j],1);
+              game.table.push(data.table[j], 1);
             }
           }
-        }
-        for (i = 0; i < removed.length; i++) {
-          for (var k = 0; k < game.table.length; k++) {
-            if (removed[i] === game.table[k].player) {
-              game.table.splice(k,1);
+          var setNotification = function () {
```

Comment:
`let` if it will get reassigned later on, or `const` if it won't.
Also, use arrow function . E.g.
```js
const setNotification = () => {
  ...
}